radar/data_management/github.yaml

```diff
@@ -0,0 +1,17 @@
+name: Github
+blip:
+  - date: 2019-05-22
+    ring: ADOPT
+description: |
```

Comment:
Good description but a bit unclear what GitHub does if you are not familiar with code